resources/js/visual-editor/editor/components/editor.css

```
@@ -1,51 +1,194 @@
+    --ve-color-primary: #0a84ff;
+    --ve-color-primary-light: #f0f7ff;
+    --ve-color-bg: #f5f5f7;
+    --ve-color-surface: #fff;
+    --ve-color-text: #1d1d1f;
+    --ve-color-text-secondary: #6e6e73;
+    --ve-color-text-muted: #888;
+    --ve-color-border: #e5e5ea;
+    --ve-color-border-strong: #d0d0d0;
+    --ve-color-dirty: #bf5af2;
+    --ve-color-error: #bf1d1d;
+    --ve-font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
```

Stylelint (17.7.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)